scipy/stats/morestats.py

```diff
@@ -2722,6 +2722,20 @@ def circmean(samples, high=2*pi, low=0, axis=None):
     -------
     circmean : float
         Circular mean.
+        
+    Examples
+    --------
+    
+    >>> from scipy.stats import morestats
```

Comment:
`from scipy.stats import circmean`: `morestats` is an implementation detail, we should not advertise it. (And while you're at it, please remove the empty line between this and `------`)